### ISSUE TITLE
Incorrect interpretation of <br> tags if no </br> tags exist (or vice versa).

### ISSUE THREAD

**Issue by trefmanic** [aruseni/bash.im-parser#2]
Hello, I tried to adapt your code to generate [fortune](https://en.wikipedia.org/wiki/Fortune_(Unix)) files instead of a database and encountered an issue. As it turns out, BeautifulSoup's **html** parser does not correctly (or maybe it's an intended feature?) interprets contents of a \<div\> if there are only opening (or only closing) br tags. That is true for bash.im, where only \<br\> tags are used. As the result, 

            quote["text"] = "".join(
                map(
                    lambda x: x if isinstance(x, unicode) else "\n",
                    text_div.contents
                )
            )

returns only line **before** the first \<br\> tag. My knowlege of Python is very limited, but there is a solution on Stackowerflow, in the question "[Beautifulsoup sibling structure with br tags](https://stackoverflow.com/questions/17639031/beautifulsoup-sibling-structure-with-br-tags)"  

In short, I had to use `soup = BeautifulSoup(html, "lxml")`

**Comment by trefmanic** [comment]
Here's a screenshot of sqlitebrowser which shows that issue:

![bash-im-parser](https://user-images.githubusercontent.com/348916/33017963-357d1c86-ce16-11e7-89e6-86c3264429a0.png)

And here what it should be (with **lxml** parser):
![bash-im-sqlite2](https://user-images.githubusercontent.com/348916/33018083-c7e7a6e0-ce16-11e7-8250-b6f698dae171.png)

**Comment by trefmanic** [comment]
Actually, this is a duplicate of an issue #1, which is marked as closed. Maybe those should be merged?

**Comment by aruseni** [comment]
Cool. Can you open a PR?

**Comment by aruseni** [comment]
BTW I can’t reproduce this issue with beautifulsoup4-4.6.0.

**Comment by aruseni** [comment]
Thanks.
